PaddleCV/object_detection/ppdet/data/data_feed.py

```diff
@@ -781,7 +781,7 @@ class SSDTestFeed(DataFeed):
 
     def __init__(self,
                  dataset=SimpleDataSet(VOC_TEST_ANNOTATION).__dict__,
-                 fields=['image'],
+                 fields=['image', 'im_id'],
```

Comment:
There is no need for SSD?

Reply:
for multiworkers will change the order of infer images, generate {im_id: im_path} dict in simple_source and get image path with im_id here, SSD train and eval use ArrageSSD and infer use ArrageTestSSD, this change is compatible with train and eval